### PR TITLE
add js2-mode attributes for better syntax highlight

### PR DESCRIPTION
**Pull request by suryatejk**

### CODE REVIEW

File: zenburn-theme.el [hunk 1]
```diff
@@ -43,10 +44,10 @@
     ("zenburn-bg-1"     . "#2B2B2B")
     ("zenburn-bg-05"    . "#383838")
     ("zenburn-bg"       . "#3F3F3F")
-    ("zenburn-bg+05"    . "#494949")
-    ("zenburn-bg+1"     . "#4F4F4F")
-    ("zenburn-bg+2"     . "#5F5F5F")
-    ("zenburn-bg+3"     . "#6F6F6F")
+    ("zenburn-bg+05" . "#282828")
```
Comment (bbatsov) [inline zenburn-theme.el]: Changing those colours is off the table.

File: zenburn-theme.el [hunk 2]
```diff
@@ -170,7 +171,7 @@ Also bind `class' to ((class color) (min-colors 89))."
    `(font-lock-constant-face ((t (:foreground ,zenburn-green+4))))
    `(font-lock-doc-face ((t (:foreground ,zenburn-green+2))))
    `(font-lock-function-name-face ((t (:foreground ,zenburn-cyan))))
-   `(font-lock-keyword-face ((t (:foreground ,zenburn-yellow :weight bold))))
+   `(font-lock-keyword-face ((t (:foreground ,zenburn-green+1 :weight bold))))
```
Comment (bbatsov) [inline zenburn-theme.el]: Changing this is off the table.

**Comment by bbatsov** [comment]
Please, don't mess with the base colours. 

Apart from this - I'll be happy to include the additional faces.

Also - commit messages should be capitalized and commits should be squashed together.

**Comment by suryatejk** [comment]
Please see my other PR, and close this.

**Pull request closed without merging**